modules/virt-connecting-vm-virtctl.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-consoles.adoc
+
+ifeval::["{context}" == "vnc-console"]
+:vnc-console:
+:console: VNC console
+endif::[]
+ifeval::["{context}" == "serial-console"]
+:serial-console:
+:console: serial console
+endif::[]
+
+:_content-type: PROCEDURE
+[id="virt-connecting-vm-virtctl_{context}"]
+= Using virtctl to connect to the {console}
+
+You can use the `virtctl` command line tool to connect to the {console} of a running virtual machine.
+
+ifdef::vnc-console[]
+[NOTE]
+====
+If you run the `virtctl vnc` command on a remote machine over an SSH connection, you must forward the X session to your local machine by running the `ssh` command with the `-X` or `-Y` flags.
+====
+
+.Prerequisites
+
+* You must install the `virt-viewer` package.
+endif::[]
+
+.Procedure
+
+* Run the following command:
++
+ifdef::serial-console[]
+[source,terminal]
+----
+$ virtctl console <vm_name>
+----
```

Comment:
It might be worth to mention that you could disconnect from the virtual machine console by typing: ctrl+].